cmd/builder/README.md

```diff
@@ -145,3 +145,20 @@ then commit the code in a git repo. A CI can sync the code and execute
 ocb --skip-generate --skip-get-modules --config=config.yaml
 ```
 to only execute the compilation step.
+
+### Strict versioning checks
+
+Strict version checking is enabled by default and checks the relevant `go.mod` 
+file for the following things after `go get`ing all components and calling 
+`go mod tidy`:
```

Comment:
Since this is here to stay, I would reword this a bit
```suggestion
The builder checks the relevant `go.mod`
file for the following things after `go get`ing all components and calling 
`go mod tidy`:
```